tests/functional/s/superfluous_parens_walrus_py38.py

```diff
@@ -1,5 +1,5 @@
 """Test the superfluous-parens warning with python 3.8 functionality (walrus operator)"""
-# pylint: disable=missing-function-docstring, invalid-name, missing-class-docstring, import-error
+# pylint: disable=missing-function-docstring, invalid-name, missing-class-docstring, import-error, pointless-statement
```

Comment:
`pointless-statement` for the walrus operator statments looks like a false positive. @clavedeluna would you be willing to open a new issue for that?


```python
(walrus := False)
print(walrus)
```
```
b.py:1:0: W0104: Statement seems to have no effect (pointless-statement)
```

Reply:
oh great point! Will open